### PR TITLE
Pre v1 cleanup - first PR

### PR DESCRIPTION
**Pull request by wpietri**
Assorted small cleanups of confusing code and unused files.

### CODE REVIEW

File: src/modelbench/benchmarks.py [hunk 1]
```diff
@@ -42,15 +42,18 @@ def __repr__(self):
...
     _hazards: list[HazardDefinition]
-    _uid_definition = {"class": "self", "version": "0.5"}
 
-    def __init__(self, hazards: List[HazardDefinition]):
+    def __init__(self):
         super().__init__()
-        self._hazards = hazards
+        self._hazards = self._make_hazards()
 
     def hazards(self) -> List[HazardDefinition]:
```
Comment (rogthefrog) [inline src/modelbench/benchmarks.py:50]: Is this a `property`?

**Review by rogthefrog — APPROVED** [review]

**Review by bkorycki — COMMENTED** [review]
Looks good! I especially like the `find_by_name` addition to benchmark definition. 
I’m curious about the choice to remove the v0.5 hazard content files because I thought we wanted to preserve v0.5 for now.

**Comment by wpietri** [comment]
> Looks good! I especially like the `find_by_name` addition to benchmark definition.

That was definitely inspired by your suggestion.

> I’m curious about the choice to remove the v0.5 hazard content files because I thought we wanted to preserve v0.5 for now.

Those files were never actually used. I believe all actual 0.5 functionality is preserved.

File: src/modelbench/benchmarks.py [hunk 2]
```diff
@@ -75,10 +78,21 @@ def __eq__(self, __value):
...
+    def _make_hazards(self) -> Sequence[HazardDefinition]:
+        pass
+
 
 class GeneralPurposeAiChatBenchmark(BenchmarkDefinition):
+    """Version 0.5 of the general purpose AI chat benchmark."""
+
+    _uid_definition = {"class": "self", "version": "0.5"}
+
     def __init__(self):
```
[the remark this answers is not in the document]
Comment (wpietri) [reply src/modelbench/benchmarks.py:91]: Could you say more? My thinking here is that we wanted UIDs for all the core objects. It gets used as part of the JSON output. Here's what the the top of that file looks like:

```
{
    "benchmark": {
        "uid": "general_purpose_ai_chat_benchmark-0.5",
```

File: src/modelbench/benchmarks.py (hunk 1 above)
Comment (wpietri) [reply src/modelbench/benchmarks.py:50]: It's currently a read-only property, yes.

File: src/modelbench/benchmarks.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (bkorycki) [reply src/modelbench/benchmarks.py:91]: Oh sorry I meant `__init__`!

**Review by bkorycki — APPROVED** [review]

**Comment by wpietri** [comment]
Added a fair bit more cleanup, this time around the SUTs.

File: src/modelbench/benchmarks.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (wpietri) [reply src/modelbench/benchmarks.py:91]: Ah, got it! Thanks for clarifying. Yes, that's totally useless and I have removed it. Well spotted.

**Comment by bkorycki** [comment]
I guess don't really understand the purpose of ModelGaugeSut. It doesn't seem to provide any functionality that doesn't already exist in the regular model gauge SUT objects.

**Review by bkorycki — APPROVED** [review]

**Pull request merged**